Makefile

```diff
@@ -18,18 +22,60 @@ clean: ## delete most git-ignored files
 	find . -name '*.pyc' -exec rm -f {} +
 	find . -name '*.pyo' -exec rm -f {} +
 	find . -name '*~' -exec rm -f {} +
+	coverage erase
```

Comment:
This is a python package,  so if you didn't install this before running the command  this will fail

Reply:
[b88f829](https://github.com/eduNEXT/eox-nelp/pull/19/commits/b88f8298a0cf44e96d25ed8b423d7a7a864888f5)